components/closebutton/index.css

```diff
@@ -18,8 +18,7 @@ governing permissions and limitations under the License.
 @import "@spectrum-css/commons/basebutton.css";
 
 .spectrum-CloseButton {
-  /* TO DO: replace with custom var during corner rounding work */
-  --spectrum-closebutton-border-radius: 9999px;
+  --spectrum-closebutton-border-radius: var(--spectrum-corner-radius-1000-full);
```

Comment:
Stumbled upon this TODO comment that probably needs to be addressed in this PR?

Using the token seems to work fine in Storybook, but the linter doesn't seem very happy about it--if I should be using a different token, I'd be happy to change this.

Reply:
Should we use `--spectrum-corner-radius-full` in this component and future components, since we're now overriding its value?:
`--spectrum-closebutton-border-radius: var(--spectrum-corner-radius-full);`

Reply:
Good callout, due to [this change](https://github.com/adobe/spectrum-css/pull/2559#discussion_r1530724751), I'll update this to `--sectrum-corner-radius-1000`.